content/en/docs/tutorials/hello-minikube.md

```diff
@@ -96,7 +96,9 @@ Pod runs a Container based on the provided Docker image.
     ```shell
     kubectl create deployment hello-node --image=registry.k8s.io/echoserver:1.4
     ```
-
+    
+    (for ARM-based deployments, use `--image=k8s.gcr.io/echoserver-arm:1.8`)
```

Comment:
Could we suggest this:
```shell
kubectl create deployment hello-node --image=registry.k8s.io/echoserver:1.8
```

I think that bump might be all that's needed. If not, isn't that a bug in the image rather than a bug in the docs?

Reply:
echoserver:1.8 does not work. However, this does:
`kubectl create deployment hello-node --image=registry.k8s.io/echoserver-arm:1.8`

Would that be preferable?

Reply:
I made the above change to this pull request.